etc/input.conf

```diff
@@ -171,6 +171,7 @@
 #l ab-loop                              # set/clear A-B loop points
 #L cycle-values loop-file "inf" "no"    # toggle infinite looping
 #ctrl+c quit 4
+#Ctrl+v loadfile ${clipboard/text} append-play; show-text 'Appended ${clipboard/text}' # append the copied path
```

Comment:
`Appended` seems quite ambiguous. Appended where? To what?

Reply:
https://github.com/mpv-player/mpv/pull/15441#discussion_r1878137646

Reply:
So maybe even shorter `+> ${clipboard/text}`

Reply:
Change to `+ ${clipboard/text}`, if the > was not a typo I don't understand what it represents.